fpa/minerselector/reptop/reptop.go

```diff
@@ -0,0 +1,43 @@
+package reptop
```

Comment:
This `fpa/minerselector/reptop` is a miner-selector algorithm that uses the Reputation Module to select the Top N miners. (`reptop` = _reputation top miners_)